internal/k8s/ako_init.go

```diff
@@ -455,8 +461,8 @@ func (c *AviController) FullSyncK8s() error {
 				for _, ingObj := range ingObjs {
 					ingLabel := utils.ObjKey(ingObj)
 					ns := strings.Split(ingLabel, "/")
-					if utils.CheckIfNamespaceAccepted(ns[0], utils.GetGlobalNSFilter(), nil, true) {
-						key := utils.Ingress + "/" + ingLabel
+					if utils.CheckIfNamespaceAccepted(ns[0], nil, true) {
```

Comment:
Why is NS filter not required anymore ?

Reply:
Now CheckIfNamespaceAccepted() function directly access NSFilter object present in same file(utils). Old code was un-necessary making an extra function call to fetch that variable(through utils.GetGlobalNSFilter()).